server/package.json

```diff
@@ -2,9 +2,11 @@
   "name": "polis",
   "version": "0.0.0",
   "description": "polis =====",
-  "main": "app.js",
+  "main": "./dist/app.js",
```

Comment:
@colinmegill here I try to express to any code consuming or calling the server that the main location is now in the dist directory that contains the  compiled JavaScript produced by the TypeScript compiler.